### PR TITLE
To-do-list structure

### PR DESCRIPTION
**Pull request by fed1k**
* Use webpack for HTML/CSS and for Javascript to to bundle
*  Basic styling

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  },
+  {
+    description: 'Going to school',
+    completed: false,
+  },
+  {
+    description: 'Washing the dishes',
+    completed: false,
+  },
+];
```
Comment (TedLivist) [inline src/index.js]: - [ ] Your tasks should have `description`, `completed` and `index` properties. Kindly add the `index` property

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  input.className = 'input';
+  input.setAttribute('type', 'checkbox');
+  const p = document.createElement('p');
+  p.className = 'listContent';
+  p.textContent = array[i].description;
+  const icon = document.createElement('i');
+  icon.className = 'fas fa-ellipsis-v';
+  inputDiv.append(input, p, icon);
+  array[i].index += 1;
+}
```
Comment (TedLivist) [inline src/index.js]: - [ ] Right now you are rendering your tasks using a **`for-loop`**. Whereas your tasks should be dynamically rendered using a function. Kindly refactor the rendering of your tasks to use a function instead.

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+## Built with:
+  * HTML
+  * Javascript
+  * CSS
+  * Webpack
+  * Fontawesome
+
+## Author
+ Follow me on: [LinkedIn](https://www.linkedin.com/in/firdavs-allamurotov-12b60a226/)  
+ Follow me on: [Github](https://github.com/fed1k)
```
Comment (TedLivist) [inline README.md]: - [ ] In your README, kindly add more detailed instructions on how users can set up and run your app locally. Note that users need to run `npm install` after cloning, before they can run the other start or build commands.

- [ ] Add the **Prerequisites** section, as we have in the [Microverse template](https://github.com/microverseinc/readme-template) declaring the technologies users need to have before the app can be run.

 - [ ] For a much more elaborate README, I would recommend you add these other sections to your documentation.

  fdk-contr

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fed1k,

Good job so far!
✔️ App is set up with webpack
✔️ HTML is configured using HTML Webpack Plugin
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (fed1k) [reply src/index.js]: @TedLivist  Hello. Thanks for explaining. But can you clarify more. Should I use function and function call should be inside loop or should I staticly call function 3 times ? Thanks for the answer in advance

Comment (TedLivist) [reply src/index.js]: @fed1k you don't really need to call the function 3 times. You can set it such that the function iterates over the tasks and builds the HTML. Then call the function once.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    description: 'Going to school',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Washing the dishes',
+    completed: false,
+    index: 0,
+  },
+];
```
Comment (keroloussamy) [inline src/index.js:19]: - [ ] Kindly the `index` property should be different from one to another, so you can use it after that to delete or update a specific task with its unique index.

**Review by keroloussamy — APPROVED** [review]
Hi @fed1k, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 4 above)
Comment (fed1k) [reply src/index.js:19]: Hi @keroloussamy. Thanks for reviewing my project. I did not get you totally. Can i send you direct message via slack to clarify your suggestion ?

Comment (keroloussamy) [reply src/index.js:19]: @fed1k Yes sure.

**Pull request merged**